detox/test/e2e/22.launch-args.test.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @detox-config-path ./detox-multiapps.config.json
```

Comment:
Albeit the coolness of this feature 😎, I'm inclined to say it even further complicates the already not-so-trivial configuration paradigm that Detox offers to users.

Also, IIUC, the end-result in the users' perspective would be that while a they might (for example) specify `android.emu.release`, they would end up getting a different configuration with the same name, which wouldn't be validated for (e.g. in terms of syntax) - or would it?
Is this really a proper dev-ex? let's discuss here...

Reply:
First, it is not a feature — it is an advanced configuration of Jest that **WE** (and not someone else) is using internally for their self-test suites. I don't add anything new to Detox itself, so it complicates one specific thing in our e2e suite, not for **users**.

Reply:
Second, all the validations are performed as usual — Detox first merges the base config from the file and the override, and then triggers all the standard validation. It indeed will be a bit different configuration, but in fact not any different from `detox test -C e2e/detox-multiapps.config.json -c android.emu.release` launch.

Reply:
@noomorph sorry for not getting to this sooner.
I think it's important that we would offer an optimization not only for ourselves, but for all devs.
Is it not likely that somewhere, developers have (or would have) 2 apps where `app 2` is really `app 1b` (i.e. different config, same binaries)?